gytheio-messaging/gytheio-messaging-benchmark/pom.xml

```diff
@@ -9,11 +9,11 @@
     </parent>
 
     <artifactId>gytheio-messaging-benchmark</artifactId>
-    
+
     <properties>
         <dependency.log4j.version>1.2.17</dependency.log4j.version>
         <dependency.camel.version>2.11.0</dependency.camel.version>
```

Comment:
Why did we not update Camel 2.11.0 here ?

Reply:
Otherwise LGTM, matches Repo master:

https://github.com/Alfresco/alfresco-repository/blob/master/pom.xml#L61-L62

        <dependency.camel.version>2.24.0</dependency.camel.version>
        <dependency.activemq.version>5.15.9</dependency.activemq.version>

Reply:
Good catch, addressed in #22.